htmresearch/regions/ColumnPoolerRegion.py

```diff
@@ -291,6 +300,7 @@ def __init__(self,
                activationThresholdDistal=13,
                connectedPermanenceDistal=0.50,
                distalSegmentInhibitionFactor=1.001,
```

Comment:
We need to update this factor's default value. You'll probably want to search the htmresearch repo and update all values. https://github.com/numenta/htmresearch/search?utf8=✓&q=distalSegmentInhibitionFactor&type=